resolwe/flow/managers/workload_connectors/kubernetes.py

```diff
@@ -495,7 +499,7 @@ def _data_inputs_size(self, data: Data, safety_buffer: int = 2 ** 30) -> int:
             .get("total_size")
         )
         assert isinstance(inputs_size, int)
-        return ceil((inputs_size + safety_buffer) / (2 ** 30)) * (2 ** 30)
+        return ceil(((inputs_size + safety_buffer) / (2 ** 30)) * 1.1) * (2 ** 30)
```

Comment:
Where do you add 2GB?

Reply:
Do you still need to round the number and then add extra space? It looks too complex; probably you can do only the latter.

Reply:
1. Safety buffer equals 2 GB by default, it can be increased.
2. Actually the requirements are specified in bytes, and my intention was to round it to the nearest gigabyte. So I add both numbers, divide to get the number of gigabytes, ceil it to the nearest integer and multiply by 2**30 again. I will make it less complicated.

Reply:
Obviously, it was too early, I haven't seen `safety_buffer` :). But still, what is the benefit of rounding it to a gigabyte? Wouldn't it work the same with the actual size?

Reply:
It most probably would. Andrej noticed that EBS size was always reported in gigabytes so I rounded it to the nearest gigabyte. But even so, 2GB of safety buffer and 10% of size should take care of any rounding problems.